rateLimiter/src/main/java/com/flipkart/varadhi/qos/RateLimiter.java

```diff
@@ -0,0 +1,11 @@
+package com.flipkart.varadhi.qos;
+
+import com.flipkart.varadhi.qos.entity.RateLimiterType;
+
+public interface RateLimiter {
+    RateLimiterType getType();
```

Comment:
I think getType is not needed here.

the caller most likely only care of isAllowed method only.

Reply:
removed it